Descension/Assets/Prefab/Items/Equippables/Arrows.prefab

```diff
@@ -44,4 +44,6 @@ MonoBehaviour:
   m_Name: 
   m_EditorClassIdentifier: 
   inventorySprite: {fileID: 21300000, guid: 2c2d6930501c33c44856e4ede5f1224e, type: 3}
+  arrowPrefab: {fileID: 7911246415520734231, guid: 223d738c21d8c174da39104ad28d6bf0, type: 3}
+  bowReticleDistance: 2
```

Comment:
Not sure why this is here. Doesn't seem like the Arrows object should be responsible for this, as it is a player configuration.

Reply:
That's an old variable that isn't in use, I just removed it in my PR which is why it's showing up here.

Reply:
```suggestion
```